src/Components/Highlight/Highlight.js

```diff
@@ -1,94 +1,109 @@
-			console.error(error);
-		}
-	};
 	showCollectionPhotos = (id) => {
-		this.setState({ display: 'block', id });
+		this.setState({ display: true, id });
 	};
 	closeCollectionPhotos = () => {
-		this.setState({ display: 'none' });
+		this.setState({ display: false, formDisplay: false });
 	};
 	componentDidUpdate(prevProps, prevState) {
-		if (this.state.page !== prevState.page) {
-			this.getPhoto();
+		if (
+			prevState.formDisplay !== this.state.formDisplay &&
+			this.state.formDisplay
+		) {
+			this.form.current.focus();
 		}
 	}
-	componentDidMount() {
-		this.getPhoto();
-	}
 	render() {
-		const { collections, isLoading, hasError, display, id } = this.state;
 		const settings = {
-			dots: true,
+			dots: false,
 			infinite: true,
 			speed: 500,
-			slidesToShow: 3,
+			slidesToShow: 5,
 			slidesToScroll: 1,
 		};
-		isLoading && <h1>Loading ....</h1>;
-		hasError && <h1>Error ....</h1>;
+		const { display, id, formDisplay } = this.state;
+		const { savedCollections } = this.props;
 		return (
 			<HighlightContainer>
 				<Container>
 					<Slider {...settings}>
-						{collections.map((collection) => (
+						<CollectionsContainer className='add'>
+							<Image
+								src={
+									'https://images.unsplash.com/photo-1513002749550-c59d786b8e6c?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxzZWFyY2h8MXx8c2t5fGVufDB8fDB8fA%3D%3D&w=1000&q=80'
+								}
+							/>
+							<AddBtn onClick={() => this.setState({ formDisplay: true })}>
+								<Add />
+							</AddBtn>
+						</CollectionsContainer>
+						{savedCollections.map((collection) => (
 							<CollectionsContainer key={collection.id}>
-								<Image
-									onClick={() => this.showCollectionPhotos(collection.id)}
-									src={collection.cover_photo.urls.small}
-									alt=''
-								/>
+								<Image src={collection.cover_photo.urls.regular} alt='' />
+								<ImageOverlay
+									onClick={() => this.showCollectionPhotos(collection.id)}>
+									<Remove
+										onClick={() =>
+											this.props.removeFromSavedCollection(collection)
+										}>
+										&times;
+									</Remove>
+								</ImageOverlay>
 							</CollectionsContainer>
 						))}
 					</Slider>
 				</Container>
-				{display === 'block' && (
-					<View display={display}>
-						<Button onClick={this.closeCollectionPhotos}>&times;</Button>
-						<HighlightContents id={id} />
+				{display && (
+					<View>
+						<Content>
+							<Remove className='close' onClick={this.closeCollectionPhotos}>
+								&times;
+							</Remove>
+							<HighlightContents id={id} />
+						</Content>
 					</View>
 				)}
+				{formDisplay && (
+					<FormContainer
+						ref={this.form}
+						onBlur={this.closeCollectionPhotos}
+						formDisplay={formDisplay}>
+						<Remove className='form' onClick={this.closeCollectionPhotos}>
+							&times;
+						</Remove>
+						<SearchFormContainer>
+							<SearchForm className='collections' formDisplay={formDisplay} />
```

Comment:
why are you using class names?

Reply:
I tried passing as a prop to use different styling on one input field, but it was not necessary